crates/holochain/src/conductor/api/api_external/admin_interface.rs

```diff
@@ -72,6 +72,29 @@ impl AdminInterfaceApi for RealAdminInterfaceApi {
                     .await?;
                 Ok(AdminResponse::AdminInterfacesAdded)
             }
+            RegisterDna(payload) => {
+                trace!(?payload);
```

Comment:
```suggestion
                trace!(register_dna_payload = ?payload);
```
Descriptive names help in understanding when reading the logs